server/audit_math/suite.py

```diff
@@ -102,9 +102,11 @@ def compute_pvalue(
         """
 
         n = self.sample_size
+        if n == 0 or reported_margin == 0:
+            return 1.0
 
-        if n == 0:
-            return 1
+        if self.sample_size == self.num_ballots:
+            return 0.0
```

Comment:
maybe just use `self.sample_size` everywhere and remove `n` for readability?

Reply:
Ah, yeah I mean to do that. Will clean up.